modules/k6/k6.go

```diff
@@ -17,17 +22,73 @@ type K6Container struct {
 	testcontainers.Container
 }
 
+type RemoteTestFileDescription struct {
```

Comment:
Wdyt if we make this name more obvious? I'd then rename https://github.com/testcontainers/testcontainers-go/pull/2350/files#diff-19a4c472d6cc0154e74ab208f0a048d900eb0e6006b4365b853d56218d73cd5fR38 to just `func downloadFile`
```suggestion
type DownloadableFile struct {
```

Reply:
Done.